 # Publish all joint states -----------------------------------
 joint_state_controller:
   type: joint_state_controller/JointStateController
   publish_rate: 50
# Velocity Controllers ---------------------------------------
 wheel_1_vel_controller:
   type: velocity_controllers/JointVelocityController
   joint: wheel_1_joint
   pid: {p: 100.0, i: 0.01, d: 10.0}
 wheel_2_vel_controller:
   type: velocity_controllers/JointVelocityController
   joint: wheel_2_joint
   pid: {p: 100.0, i: 0.01, d: 10.0}
 wheel_3_vel_controller:
   type: velocity_controllers/JointVelocityController
   joint: wheel_3_joint
   pid: {p: 100.0, i: 0.01, d: 10.0}
 wheel_4_vel_controller:
   type: velocity_controllers/JointVelocityController
   joint: wheel_4_joint
   pid: {p: 100.0, i: 0.01, d: 10.0}
 wheel_5_vel_controller:
   type: velocity_controllers/JointVelocityController
   joint: wheel_5_joint
   pid: {p: 100.0, i: 0.01, d: 10.0}
 wheel_6_vel_controller:
   type: velocity_controllers/JointVelocityController
   joint: wheel_6_joint
   pid: {p: 100.0, i: 0.01, d: 10.0}

